cardano-api/cardano-api.cabal

```diff
@@ -1,3 +1,5 @@
+cabal-version: 2.4
```

Comment:
Probably should be `3.0` because we are using `3.0` features (ie the `cabal.project` file).

Reply:
I tried `3.0` but it failed the build.  For some reason `2.4` is as high as we can go.

Reply:
Ok, I would be fine leaving it like that for now. The required nix fix is probably large and hairy.